src/orchestration/Orchestration.ts

```diff
@@ -387,6 +422,40 @@ export class Orchestration {
         this.eventCache.recordEvent(eventType, eventData);
     }
 
+    /**
+     * Loads evaluations for a given list of features using the provided user context and ID.
+     *
+     * @param request An object with the user indentification and context information and a list of requested features.
+     */
+    public loadEvaluations(request: EvidentlyRequest) {
+        if (request.features.length > MAX_EVIDENTLY_FEATURES_PER_EVENT) {
+            throw new Error(
+                `Can only request up to ${MAX_EVIDENTLY_FEATURES_PER_EVENT} features at a time`
+            );
+        }
```

Comment:
suggestion: Have the EvidentlyManager do the check and throw this error. By moving the business logic to EvidentlyManager, Orchestration becomes more maintainable (i.e., separation of concerns).